examples/auth_demo/auth_demo/router.py

```diff
@@ -56,7 +58,8 @@ async def status(
 @router.get("/reception")
 @inject
 async def reception(
-    auth_context: AuthContext = get_auth, hangout: HangoutPort = provide_hangout
+    auth_context: Optional[DemoAuthContext] = get_auth,
+    hangout: HangoutPort = provide_hangout,
```

Comment:
I think for readability, it might be better to spell this out:
```suggestion
    hangout: HangoutPort =  Depends(Provide[Container.hangout]),
```

Reply:
Ok - you need to approve again.